java/src/jmri/jmrit/display/layoutEditor/LayoutEditorDialogs/Bundle.java

```diff
@@ -0,0 +1,100 @@
+package jmri.jmrit.display.layoutEditor.LayoutEditorDialogs;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.*;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @since 3.3.1
+ */
+public class Bundle extends jmri.jmrit.display.layoutEditor.Bundle {
+
+    @CheckForNull
+    private static final String name = "jmri.jmrit.display.layoutEditor.LayoutEditorBundle"; // NOI18N
```

Comment:
The bundle this file refers to needs to be in the package the file resides in.  If it isn’t. The name needs to be set to null.

The whole bundle system is designed so that if a key isn’t found in the local bundle you look to the parent bundle, but not following the rules bypasses the tree structure.

Reply:
Resource package file name set to null. Thanks for the review & catch.